File: 백준/Bronze/30087. 진흥원 세미나/진흥원 세미나.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        for (int i = 0; i < n; i++)
        {
            String s = sc.next();

            if (s.equals("Algorithm"))
                System.out.println(204);
            else if (s.equals("DataAnalysis"))
                System.out.println(207);
            else if (s.equals("ArtificialIntelligence"))
                System.out.println(302);
            else if (s.equals("CyberSecurity"))
                System.out.println("B101");
            else if (s.equals("Network"))
                System.out.println("303");
            else if (s.equals("Startup"))
                System.out.println("501");
            else if (s.equals("TestStrategy"))
                System.out.println("105");
        }
    }
}

File: 백준/Bronze/30087. 진흥원 세미나/진흥원 세미나.cs
int N = int.Parse(Console.ReadLine());

for (int i = 0; i < N; i++)
{
    string s = Console.ReadLine();

    if (s == "Algorithm")
        Console.WriteLine(204);
    else if (s == "DataAnalysis")
        Console.WriteLine(207);
    else if (s == "ArtificialIntelligence")
        Console.WriteLine(302);
    else if (s == "CyberSecurity")
        Console.WriteLine("B101");
    else if (s == "Network")
        Console.WriteLine("303");
    else if (s == "Startup")
        Console.WriteLine("501");
    else if (s == "TestStrategy")
        Console.WriteLine("105");
}